tck/src/main/java/org/eclipse/microprofile/graphql/tck/apps/superhero/db/UnknownTeamException.java

```diff
@@ -21,4 +21,5 @@
     public UnknownTeamException(String message) {
         super(message);
     }
+
```

Comment:
Same as above - I'd remove this whitespace-only change to avoid noise.

Reply:
+1